applications/dashboard/views/role/edit.php

```diff
@@ -1,10 +1,4 @@
-<?php if (!defined('APPLICATION')) exit(); ?>
```

Comment:
Yes, I've dropped it because I thought I remembered a discussion stating that this is not needed any more. Frankly spoken, I do not understand the difference why it is not needed in classes but needed in views.
But nevertheless, I've corrected the PR